### ISSUE TITLE
unsupported Java version "16", defaulting to 1.7 [ERROR] Internal error...

### ISSUE THREAD

**Issue by AndyObtiva** [ruby/psych#520]
- MacOS Big Sur 11.2.3 x64
- Processor 2.5 GHz Quad-Core Intel Core i7 x64
- rvm 1.29.12-next
- java version "16.0.2" 2021-07-20 Java(TM) SE Runtime Environment (build 16.0.2+7-67) Java HotSpot(TM) 64-Bit Server VM (build 16.0.2+7-67, mixed mode, sharing)
- jruby 9.2.19.0 (2.5.8) 2021-06-15 55810c552b Java HotSpot(TM) 64-Bit Server VM 16.0.2+7-67 on 16.0.2+7-67 +jit [darwin-x86_64]
- gem version 3.2.27 (after running `gem update --system`)

```
andymaleh@Andys-MacBook-Pro snowboard_utah % gem install psych -v4.0.1 
Fetching psych-4.0.1-java.gem
  jar dependencies for psych-4.0.1-java.gemspec . . .
unsupported Java version "16", defaulting to 1.7
[ERROR] Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @4d96250 -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @4d96250
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:121)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @4d96250
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/InternalErrorException
ERROR:  While executing gem ... (Errno::ENOENT)
    No such file or directory - /Users/andymaleh/.rvm/gems/jruby-9.2.19.0@snowboard_utah/gems/psych-4.0.1-java/deps.lst
```

Help would be greatly appreciated!

**Comment by AndyObtiva** [comment]
Just a note, it happens on Linux too.

jruby 9.2.19.0 (2.5.8) 2021-06-15 55810c552b Java HotSpot(TM) 64-Bit Server VM 16.0.2+7-67 on 16.0.2+7-67 +jit [linux-x86_64]

```
$ gem install psych
Fetching psych-4.0.1-java.gem
  jar dependencies for psych-4.0.1-java.gemspec . . .
Installing gem 'ruby-maven' . . .
Fetching ruby-maven-libs-3.3.9.gem
Fetching ruby-maven-3.3.12.gem
Successfully installed ruby-maven-libs-3.3.9
Successfully installed ruby-maven-3.3.12

using maven for the first time results in maven
downloading all its default plugin and can take time.
as those plugins get cached on disk and further execution
of maven is much faster then the first time.

unsupported Java version "16", defaulting to 1.7
[ERROR] Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @506dcf55 -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @506dcf55
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:121)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @506dcf55
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/InternalErrorException
ERROR:  While executing gem ... (Errno::ENOENT)
    No such file or directory - /home/oem/.rvm/gems/jruby-9.2.19.0@snowboard_utah/gems/psych-4.0.1-java/deps.lst
```

**Comment by AndyObtiva** [comment]
I just tried it with jruby-9.3.0.0 on the same Mac x64 machine. Same issue.

jruby 9.3.0.0 (2.6.8) 2021-09-17 85c20e780f Java HotSpot(TM) 64-Bit Server VM 16.0.2+7-67 on 16.0.2+7-67 +jit [darwin-x86_64]

```
% gem install psych 
Fetching psych-4.0.1-java.gem
  jar dependencies for psych-4.0.1-java.gemspec . . .
unsupported Java version "16", defaulting to 1.7
[ERROR] Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @138caeca -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @138caeca
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:121)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @138caeca
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/InternalErrorException
ERROR:  While executing gem ... (Errno::ENOENT)
    No such file or directory - /Users/andymaleh/.rvm/gems/jruby-9.3.0.0@glimmer-dsl-swt/gems/psych-4.0.1-java/deps.lst
```

**Comment by olleolleolle** [comment]
@headius is this about opens-flags needed to be passed somewhere?

**Comment by AndyObtiva** [comment]
Hi, 

I just wanted to note that in the newly released JRuby 9.3.3.0, I still get into an issue with ruby-maven when installing psych v4.0.3:

```shell
using maven for the first time results in maven
downloading all its default plugin and can take time.
as those plugins get cached on disk and further execution
of maven is much faster then the first time.

[DEPRECATED] `Bundler.with_clean_env` has been deprecated in favor of `Bundler.with_unbundled_env`. If you instead want the environment before bundler was originally loaded, use `Bundler.with_original_env` (called at /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/ruby-maven-3.3.12/lib/ruby_maven.rb:38)
Using warbler 2.0.5
unsupported Java version "17", defaulting to 1.7
[ERROR] Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @34819867 -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @34819867
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:121)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) library `java' could not be loaded: java.lang.reflect.InaccessibleObjectException: Unable to make protected native java.lang.Object java.lang.Object.clone() throws java.lang.CloneNotSupportedException accessible: module java.base does not "opens java.lang" to unnamed module @34819867
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/InternalErrorException
--- ERROR REPORT TEMPLATE -------------------------------------------------------

Errno::ENOENT: No such file or directory - /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/psych-4.0.3-java/deps.lst
  org/jruby/RubyIO.java:1227:in `sysopen'
  org/jruby/RubyIO.java:3986:in `read'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/jars/installer.rb:69:in `load_from_maven'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/jars/installer.rb:235:in `install_dependencies'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/jars/installer.rb:224:in `do_install'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/jars/installer.rb:170:in `vendor_jars'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/jars/post_install_hook.rb:28:in `block in <main>'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/rubygems/installer.rb:367:in `block in run_post_install_hooks'
  org/jruby/RubyArray.java:1865:in `each'
  /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/stdlib/rubygems/installer.rb:366:in `run_post_install_hooks'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/rubygems_gem_installer.rb:43:in `install'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/source/rubygems.rb:204:in `install'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/installer/gem_installer.rb:54:in `install'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/installer/gem_installer.rb:16:in `install_from_spec'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/installer/parallel_installer.rb:186:in `do_install'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/installer/parallel_installer.rb:177:in `block in worker_pool'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/worker.rb:62:in `apply_func'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/worker.rb:57:in `block in process_queue'
  org/jruby/RubyKernel.java:1507:in `loop'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/worker.rb:54:in `process_queue'
  /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/gems/bundler-2.3.5/lib/bundler/worker.rb:91:in `block in create_threads'

## Environment

Bundler             2.3.5
  Platforms         ruby, universal-java-17
Ruby                2.6.8p0 (2022-01-19 revision 67951) [java]
  Full Path         /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/bin/jruby
  Config Dir        /Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/etc
RubyGems            3.2.29
  Gem Home          /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt
  Gem Path          /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt:/Users/andymaleh/.rvm/rubies/jruby-9.3.3.0/lib/ruby/gems/shared
  User Home         /Users/andymaleh
  User Path         /Users/andymaleh/.gem/jruby/2.6.0
  Bin Dir           /Users/andymaleh/.rvm/gems/jruby-9.3.3.0@glimmer-dsl-swt/bin
OpenSSL             
  Compiled          JRuby-OpenSSL 0.11.0
  Loaded            JRuby-OpenSSL 0.11.0
  Cert File         /Library/Java/JavaVirtualMachines/jdk-17.0.1.jdk/Contents/Home/lib/security/cacerts
  Cert Dir          /etc/ssl/certs
Tools               
  Git               2.24.3 (Apple Git-128)
  RVM               1.29.12-next (master)
  rbenv             not installed
  chruby            not installed
  rubygems-bundler  (1.4.5)

## Bundler Build Metadata

Built At          2022-01-12
Git SHA           a13d015fcb
Released Version  true

## Gemfile

### Gemfile

source 'http://rubygems.org'

ruby '>= 2.6.0', :engine => 'jruby', engine_version: '>= 9.3.0.0'

# Glimmer project owner gems
gem 'glimmer', '~> 2.5.5'
gem 'super_module', '>= 1.4.1', '< 2.0.0'
gem 'nested_inherited_jruby_include_package', '>= 0.3.0', '< 2.0.0'
gem 'puts_debuggerer', '>= 0.13.0', '< 2.0.0', require: false
gem 'rake-tui', '>= 0.2.3', '< 2.0.0'

# 3rd party gems (flexible versions to support other user gems that may share dependencies)
gem 'concurrent-ruby', '>= 1.1.7', '< 2.0.0'
gem 'jruby-win32ole', '>= 0.8.5', '< 2.0.0'
gem 'os', '>= 1.0.0', '< 2.0.0'
gem 'rake', '>= 13.0.0'
gem 'text-table', '>= 1.2.4', '< 2.0.0'
gem 'rouge', '>= 3.26.0', '< 4.0.0'

group :development do
  gem 'juwelier', '>= 2.4.9', '< 3.0.0'
  gem 'warbler', '>= 2.0.5', '< 3.0.0'
  gem 'rspec-mocks', '~> 3.0'
  gem 'rspec', '~> 3.0'
  gem 'coveralls', '= 0.8.23', require: false
  gem 'simplecov', '~> 0.16.1', require: nil
  gem 'simplecov-lcov', '~> 0.7.0', require: nil
end

### Gemfile.lock

GEM
  remote: http://rubygems.org/
  specs:
    addressable (2.8.0)
      public_suffix (>= 2.0.2, < 5.0)
    array_include_methods (1.4.0)
    awesome_print (1.9.2)
    builder (3.2.4)
    concurrent-ruby (1.1.9)
    coveralls (0.8.23)
      json (>= 1.8, < 3)
      simplecov (~> 0.16.1)
      term-ansicolor (~> 1.3)
      thor (>= 0.19.4, < 2.0)
      tins (~> 1.6)
    descendants_tracker (0.0.4)
      thread_safe (~> 0.3, >= 0.3.1)
    diff-lcs (1.4.4)
    docile (1.4.0)
    facets (3.1.0)
    faraday (1.8.0)
      faraday-em_http (~> 1.0)
      faraday-em_synchrony (~> 1.0)
      faraday-excon (~> 1.1)
      faraday-httpclient (~> 1.0.1)
      faraday-net_http (~> 1.0)
      faraday-net_http_persistent (~> 1.1)
      faraday-patron (~> 1.0)
      faraday-rack (~> 1.0)
      multipart-post (>= 1.2, < 3)
      ruby2_keywords (>= 0.0.4)
    faraday-em_http (1.0.0)
    faraday-em_synchrony (1.0.0)
    faraday-excon (1.1.0)
    faraday-httpclient (1.0.1)
    faraday-net_http (1.0.1)
    faraday-net_http_persistent (1.2.0)
    faraday-patron (1.0.0)
    faraday-rack (1.0.0)
    git (1.9.1)
      rchardet (~> 1.8)
    github_api (0.19.0)
      addressable (~> 2.4)
      descendants_tracker (~> 0.0.4)
      faraday (>= 0.8, < 2)
      hashie (~> 3.5, >= 3.5.2)
      oauth2 (~> 1.0)
    glimmer (2.5.5)
      array_include_methods (~> 1.4.0)
      facets (>= 3.1.0, < 4.0.0)
    hashie (3.6.0)
    highline (2.0.3)
    jar-dependencies (0.4.1)
    jruby-jars (9.3.1.0)
    jruby-rack (1.1.21)
    jruby-win32ole (0.8.5)
    json (2.6.1-java)
    juwelier (2.4.9)
      builder
      bundler
      git
      github_api
      highline
      kamelcase (~> 0)
      nokogiri
      psych
      rake
      rdoc
      semver2
    jwt (2.3.0)
    kamelcase (0.0.2)
      semver2 (~> 3)
    method_source (1.0.0)
    multi_json (1.15.0)
    multi_xml (0.6.0)
    multipart-post (2.1.1)
    nested_inherited_jruby_include_package (0.3.0)
    nokogiri (1.12.5-java)
      racc (~> 1.4)
    oauth2 (1.4.7)
      faraday (>= 0.8, < 2.0)
      jwt (>= 1.0, < 3.0)
      multi_json (~> 1.3)
      multi_xml (~> 0.5)
      rack (>= 1.2, < 3)
    os (1.1.4)
    pastel (0.8.0)
      tty-color (~> 0.5)
    psych (3.3.2-java)
      jar-dependencies (>= 0.1.7)
    public_suffix (4.0.6)
    puts_debuggerer (0.13.1)
      awesome_print (~> 1.9.2)
    racc (1.6.0-java)
    rack (2.2.3)
    rake (13.0.6)
    rake-tui (0.2.3)
      tty-prompt
    rchardet (1.8.0)
    rdoc (6.3.3)
    rouge (3.26.1)
    rspec (3.10.0)
      rspec-core (~> 3.10.0)
      rspec-expectations (~> 3.10.0)
      rspec-mocks (~> 3.10.0)
    rspec-core (3.10.1)
      rspec-support (~> 3.10.0)
    rspec-expectations (3.10.1)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.10.0)
    rspec-mocks (3.10.2)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.10.0)
    rspec-support (3.10.3)
    ruby2_keywords (0.0.5)
    rubyzip (1.3.0)
    semver2 (3.4.2)
    simplecov (0.16.1)
      docile (~> 1.1)
      json (>= 1.8, < 3)
      simplecov-html (~> 0.10.0)
    simplecov-html (0.10.2)
    simplecov-lcov (0.7.0)
    super_module (1.4.1)
      method_source (>= 0.8.2, < 1.1.0)
    sync (0.5.0)
    term-ansicolor (1.7.1)
      tins (~> 1.0)
    text-table (1.2.4)
    thor (1.1.0)
    thread_safe (0.3.6-java)
    tins (1.29.1)
      sync
    tty-color (0.6.0)
    tty-cursor (0.7.1)
    tty-prompt (0.23.1)
      pastel (~> 0.8)
      tty-reader (~> 0.8)
    tty-reader (0.9.0)
      tty-cursor (~> 0.7)
      tty-screen (~> 0.8)
      wisper (~> 2.0)
    tty-screen (0.8.1)
    warbler (2.0.5)
      jruby-jars (>= 9.0.0.0)
      jruby-rack (>= 1.1.1, < 1.3)
      rake (>= 10.1.0)
      rubyzip (~> 1.0, < 1.4)
    wisper (2.0.1)

PLATFORMS
  universal-java-1.8
  universal-java-16
  universal-java-17

DEPENDENCIES
  concurrent-ruby (>= 1.1.7, < 2.0.0)
  coveralls (= 0.8.23)
  glimmer (~> 2.5.5)
  jruby-win32ole (>= 0.8.5, < 2.0.0)
  juwelier (>= 2.4.9, < 3.0.0)
  nested_inherited_jruby_include_package (>= 0.3.0, < 2.0.0)
  os (>= 1.0.0, < 2.0.0)
  puts_debuggerer (>= 0.13.0, < 2.0.0)
  rake (>= 13.0.0)
  rake-tui (>= 0.2.3, < 2.0.0)
  rouge (>= 3.26.0, < 4.0.0)
  rspec (~> 3.0)
  rspec-mocks (~> 3.0)
  simplecov (~> 0.16.1)
  simplecov-lcov (~> 0.7.0)
  super_module (>= 1.4.1, < 2.0.0)
  text-table (>= 1.2.4, < 2.0.0)
  warbler (>= 2.0.5, < 3.0.0)

RUBY VERSION
   ruby 2.6.8p0 (jruby 9.3.1.0)

BUNDLED WITH
   2.3.5

--- TEMPLATE END ----------------------------------------------------------------

Unfortunately, an unexpected error occurred, and Bundler cannot continue.

First, try this link to see if there are any existing issue reports for this error:
https://github.com/rubygems/rubygems/search?q=No+such+file+or+directory+-+%2FUsers%2Fandymaleh%2F.rvm%2Fgems%2Fjruby-9.3.3.0%40glimmer-dsl-swt%2Fgems%2Fpsych-4.0.3-java%2Fdeps.lst&type=Issues

If there aren't any reports for this error yet, please fill in the new issue form located at https://github.com/rubygems/rubygems/issues/new?labels=Bundler&template=bundler-related-issue.md, and copy and paste the report template above in there.
```

I know that [this Pull Request](https://github.com/jruby/ruby-maven/pull/2) (https://github.com/jruby/ruby-maven/pull/2) fixed the issue. Will there be a new release of ruby-maven soon?

**Comment by headius** [comment]
The new ruby-maven 3.3.13 has been released with the fix mentioned above, which should fix this issue and jruby/jruby#7026. Sorry for the delay!
